config/example_artillery_genius_pro_2023.cfg

```diff
@@ -0,0 +1,153 @@
+[include mainsail.cfg]
```

Comment:
Please do not put custom includes in stock files, the end user may not have mainsail